_benchmark/commands/command-flags.md

```diff
@@ -0,0 +1,204 @@
+
+## contender
+
+The TestExecution ID for the contender being compared to the baseline. 
+
+## results-format
+
+Defines the output format for the command line results, either `markdown` or `csv`. Default is `markdown`.
+
+## results-number-align
+
+Defines the column number alignment for when the `compare` command outputs results. Default is `right`.
+
+## results-file
+
+When provided a file path, writes the compare results to the file indicated in the path. 
+
+## show-in-results
+
+Determines whether or not to include the comparison in the results file. 
+
+## provision-config-repository
+
+Defines the repository from which OpenSearch Benchmark loads `provision-configs` and `provision-config-instances`. 
+
+## provision-config-revision
+
+Defines a specific Git revision in the `provision-config` that OpenSearch Benchmark should use. 
+
+## provision-config-path
+
+Defines the path to the `--provision-config-instance` and any OpenSearch plugin configurations to use. 
+
+## distribution-version
+
+Downloads the specified OpenSearch distribution based on version number. For a list of released OpenSearch versions, see [Version history](https://opensearch.org/docs/version-history/).
+
+## distribution-repository
+
+Defines the repository from where the OpenSearch distribution should be downloaded. Default is `release`.
+
+## provision-config-instance
+
+Defines the `--provision-config-instance` to use. You can view possible configuration instances using the command `opensearch-benchmark list provision-config-instances`.  
+
+## provision-config-instance-params
+
+A comma-separated list of key-value pairs injected verbatim as variables for the `provision-config-instance`.
+
+## target-hosts
+
+Defines a comma-separated list of host-port pairs that should be targeted if using the pipeline `benchmark-only`. Default is `localhost:9200`. 
+
+## target-os
+
+The target operating system (OS) for which the OpenSearch artifact should be downloaded. Default is the current OS.
+
+## target-arch
+
+The name of the CPU architecture for which an artifact should be downloaded. 
+
+## revision
+
+Defines the current source code revision to use for running a benchmark test. Default is `current`.
+
+This command flag can use the following options:
+
+   - `current`: Uses the source tree's current revision based on your OpenSearch distribution. 
+   - `latest`: Fetches the latest revision from the main branch of the source tree. 
+   - You can also use a timestamp or commit ID from the source tree. When using a timestamp, specify `@ts`, where "ts" is a valid ISO 8601 timestamp, for example, `@2013-07-27T10:37:00Z`. 
+
+## opensearch-plugins
+
+Defines which [OpenSearch plugins]({{site.url}}{{site.baseurl}}/install-and-configure/plugins/) to install. By default, no plugins are installed.
+
+## plugin-params
+
+Defines a comma-separated list of key:value pairs that are injected verbatim into all plugins as variables.
```

Comment:
See line 112 for key-value pair. One way or the other. But I think "key-value" pair is more prevalent.